data_warehouse/decorators/context_params_check_decorators.py

```diff
@@ -0,0 +1,42 @@
+'''context参数检查装饰器模块'''
+from infra.exceptions import BadRequest
+import functools
+
+
+def __check_request(args):
+    if args[-1] is None:
+                raise BadRequest('请给定context参数。')
```

Comment:
BTY, I think the error messages are for developers, since these errors are from client side, instead they are introduced by our programers, maybe you should log this error in prod logger with level error, and raise 5XX error